### PR TITLE
Update theurgy.lic

### PR DESCRIPTION
**Pull request by drboldwin**
Having both tithe's in there really does us no good since there is a 10 minute timer (so just dropping coin for nothing) same likely with crossing-training tithe.  I kept the one closer to the bank/what comes next.

Also Gabellia reported she didn't have enough coin on hand for supplies - I couldn't find where that line went so added it in a spot somewhat arbitrarily for now - this could go in the check_supplies method if we want.

### CODE REVIEW

**Comment by rpherbig** [comment]
Why reduce the amount of tithing?

**Comment by drboldwin** [comment]
Hehe you were too fast - didn't finish writing up why yet :-P

File: theurgy.lic [hunk 1]
```diff
@@ -476,6 +474,7 @@ class Theurgy
     ######################## SCRIPT ########################
 
     pray
+    ensure_copper_on_hand(600, $settings.hometown)
```
Comment (rpherbig) [inline theurgy.lic]: This should be `@hometown`?

**Comment by rpherbig** [comment]
Interesting. https://elanthipedia.play.net/Almsbox suggests the max is 9 silver, both https://elanthipedia.play.net/Clerical_rituals#Release_from_greed and https://elanthipedia.play.net/Soul_system#Improving_soul_state suggest the max is 5.

Please update `crossing-training` with this PR.

File: theurgy.lic (hunk 1 above)
Comment (drboldwin) [inline theurgy.lic]: touche'

**Comment by drboldwin** [comment]
yeah I had read 5 and rolled with that.  Never saw the other one

**Comment by rpherbig** [comment]
The timer is different between Cleric and Paladin, that's kind of annoying...

**Comment by drboldwin** [comment]
is it?  That is odd.  I suppose we should use whichever has the shorter timer then

File: theurgy.lic [hunk 2]
```diff
@@ -476,6 +474,7 @@ class Theurgy
     ######################## SCRIPT ########################
 
     pray
+    ensure_copper_on_hand(600, @hometown)
```
Comment (rpherbig) [inline theurgy.lic]: I suggest moving this into check_supplies

**Comment by rpherbig** [comment]
That's what the two links said (10 minutes vs 4 hours). It seems like a pretty dramatic difference to me, but I haven't tested it.

We can set the cooldown based on the class (DRStats.cleric? and/or DRStats.paladin?) if we wanted to.

File: theurgy.lic (hunk 2 above)
Comment (drboldwin) [inline theurgy.lic]: I figured that would be the case.  Just wanted to get the pull request in first and figured could move it after.

**Comment by drboldwin** [comment]
I don't know that it matters which you tithe to - as I cleric I get the same devotion from the cleric one as I do the paladin one.  So we should just force pally's to fund the cleric guild

**Comment by rpherbig** [comment]
Sorry, I wasn't suggesting we use guild to determine *where* to tithe, but rather the *cooldown*.

**Comment by drboldwin** [comment]
Ahh I get it - I had misunderstood and thought you meant the cleric box and paladin box had different cooldowns - not the guilds themselves had different cooldowns.

**Comment by drboldwin** [comment]
```
    walk_to(shrine['start_room'])
    shrine['path_in'].each { |dir| move(dir) }
  end
```

can we change this to an each do with a fixstanding? check called between each?  Will help to solve the low athletics people falling over.

**Comment by rpherbig** [comment]
We need to get those people moved over to the lich fork :)

**Comment by drboldwin** [comment]
Oh bah I keep forgetting that's the issue here.

**Comment by drboldwin** [comment]
What were we still trying to do here?

**Comment by rpherbig** [comment]
The timer.

**Comment by drboldwin** [comment]
Well you can push this then.  Timer is in crossing Training I think.  My fork is likely behind on that one, so I'll delete it and refork but will have to open up a new Pull Request at that point.

**Pull request merged**